repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasRelationshipStoreV2.java

```diff
@@ -408,28 +424,33 @@ public AtlasRelationship getOrCreate(AtlasRelationship relationship) throws Atla
 
         // check if relationship exists
         AtlasEdge relationshipEdge = getRelationship(end1Vertex, end2Vertex, relationship);
-
+        boolean isCreated = false;
```

Comment:
what is `isCreated` referring to ? can you rename to be specific

Reply:
old PR